tests/MessageHubTests/src/test/java/com/ibm/streamsx/messagehub/test/MessageHubAppConfigParamTest.java

```diff
@@ -82,7 +82,7 @@ public void messageHubAppConfigParamTest() throws Exception {
 		StreamsContext<?> context = StreamsContextFactory.getStreamsContext(Type.DISTRIBUTED_TESTER);
 		Tester tester = topo.getTester();
 		Condition<List<String>> condition = MessageHubSPLStreamsUtils.stringContentsUnordered(tester, msgStream, Constants.STRING_DATA);
-		tester.complete(context, condition, 30, TimeUnit.SECONDS);
+		tester.complete(context, new HashMap<>(), condition, 30, TimeUnit.SECONDS);
```

Comment:
It prevents the test cases to fail. Same bug as https://github.com/IBMStreams/streamsx.kafka/issues/25.